Packs/CortexAttackSurfaceManagement/Scripts/SnmpDetection/SnmpDetection.yml

```diff
@@ -0,0 +1,25 @@
+args:
+- description: IP address.
+  name: ip_address
+  required: true
+- default: true
+  defaultValue: "3"
+  description: Time out.
```

Comment:
please update the description to:
`Request timeout value, in seconds.`
or minutes, verify it and update